retworkx/__init__.py

```diff
@@ -535,6 +555,9 @@ def is_isomorphic_node_match(first, second, matcher):
         one for each node data object. If the return of this
         function evaluates to True then the nodes passed to it are vieded
         as matching.
+    :param bool (default=True) id_order:  If set to true, the algorithm matches
+        the nodes in order specified by their ids. Otherwise, it uses a
+        heuristic matching order based in [VF2]_ paper.
```

Comment:
```suggestion
    :param bool id_order: If set to ``False`` this function will use a
        heuristic matching order based on [VF2]_ paper. Otherwise it will
        default to matching the nodes in order specified by their ids.
```